10.u-root/README.md

```diff
@@ -0,0 +1,769 @@
+# u-root, what it is and why it is
```

Comment:
Some work is needed to bring this up to date and formatted correctly. I feel like we can submit this now and work through the issues in a separate PRs.